Pipfile

```diff
@@ -32,6 +32,9 @@ graphql-core = ">=2.1,<3"
 pytz = ">=2017.2"
 six = ">=1.10.0"
 sqlalchemy = ">=1.3.0,<2"
+neo4j = "==1.7.4"
```

Comment:
Are these packages necessary when simply using the compiler, or are they only used in tests? If the latter, move them to the dev section above.

Reply:
Good point-- done